tests/hijack/http.js

```diff
@@ -24,3 +25,47 @@ addAsyncTest('HTTP - meteor/http - call a server', async function (test) {
   test.stableEqual(events, expected);
   test.equal(result, 200);
 });
+
+const server = http.createServer((req, res) => {
+  res.end('done');
+}).listen();
+
+addAsyncTest('HTTP - node:http - client request', async function (test) {
```

Comment:
Makes sense

Reply:
Yeah, it does pollute the current context and leaks even when both are awaited... having a hard time thinking on a variable or parameter to use to prevent the duplication.

Reply:
I tried using a separate store with `run` and it worked fine. I think we can use `Meteor.Environment` on Meteor 2.